README.md

```diff
@@ -1,72 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello_Microverse
 
-> Description the project.
+> A simple project to show how linters, GitHub workflow and branches work
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+- HTML5
+- CSS3
+- VSCode or any source code editor of your choice
+- GitHub
+- Terminal
 
 
 ## Authors
 
 👤 **Author1**
```

Comment:
- [ OPTIONAL] Kindly replace the generic `Author1` text to your name - the author of this repository :+1: .